### DISCUSSION TITLE
Embed a PDF in a figure

### DISCUSSION THREAD

**Discussion by fbanning** [typst/typst#224] [Q&A] [answered]
I normally use PDFs and SVGs for the figures in my papers. How can I add those as the image in a figure environment? I've tried the following but it throws an error that PDF is an unsupported format:

```
#figure(
    image("Figures/my_figure.pdf"),
    caption: [Nice caption goes here.],
) <my-figure>
```

**Comment by laurmaedje** [comment, marked as answer]
PDF is unfortunately not yet supported, svg should just work though.

**Comment by fbanning** [comment]
Ah, cool, thanks for the heads-up. I'll convert the relevant pieces to SVG then and try it out with like that.

I know you have all your hands full with stuff to do right now but if you'd be interested in adding PDF support as well in the long run, that would make me quite happy. :)
